L1Trigger/L1CaloTrigger/python/test_L1EGCrystalClusterEmulator_cfg.py

```diff
@@ -0,0 +1,76 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process("L1AlgoTest",eras.Phase2_trigger)
```

Comment:
repeated comment:

please use the actual era for the phase 2 workflow being tested
(phase2_trigger is an internal use modifier, Phase2_trigger with that capitalization is not anything)

"please use the actual era for the phase 2 workflow being tested" does not mean "use the Phase2 era"

different scenarios have different associated eras, seen in https://github.com/cms-sw/cmssw/blob/f18f5a71e7278be55d457d17e38379b11cd04c19/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L740

Reply:
done.